_includes/header.html

```diff
@@ -21,6 +21,7 @@ <h3><a href="/">FRC Team 1418 - George Mason High School</a></h3>
             <li><a href="/">Home</a></li>
             <li><a href="/robots">Robots</a>
                 <ul>
+                    <li><a href="/robot/2020">2020</a></li>
```

Comment:
Not an issue with this PR but it strikes me that we could probably automate making the list of robot pages